static/app/views/replays/replays.tsx

```diff
@@ -106,11 +117,11 @@ function Replays(props: Props) {
             </Link>
           }
           user={{
-            username: replay['user.display'],
-            id: replay['user.display'],
-            ip_address: replay['user.display'],
-            name: replay['user.display'],
-            email: replay['user.display'],
+            username: replay['user.username'] ?? '',
+            id: replay['user.id'] ?? '',
+            ip_address: replay['user.ip_address'] ?? '',
+            name: replay['user.name'] ?? '',
+            email: replay['user.email'] ?? '',
```

Comment:
before the generated avatar image was something like:
<img width="47" alt="Screen Shot 2022-04-28 at 3 26 47 PM" src="https://user-images.githubusercontent.com/187460/165857906-42b64021-49e5-448f-8e9c-c091784e45dc.png">

The color and digits are based on the first digit in the ip address :(

Now we pass in any user data that we have fetched, including the ip_address. Now if only the `ip_address` field is present (this is the case for all our test data) the avatar image will instead always be:
<img width="51" alt="Screen Shot 2022-04-28 at 3 28 49 PM" src="https://user-images.githubusercontent.com/187460/165858143-2ecd03b3-9e73-4c12-9a1d-c2e35f463419.png">

This is to match with the fields we have in the details page, so the avatar images are the same.